packages/nssm/windows.buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "sources": {
     "nssm": {
       "kind": "url",
-      "url": "https://nssm.cc/release/nssm-2.24.zip",
-      "sha1": "be7b3577c6e3a280e5106a9e9db5b3775931cefc"
+      "url": "http://nssm.cc/ci/nssm-2.24-101-g897c7ad.zip",

```

Comment:
@vadymzagorodko What is the reason behind moving from a stable version to a ci artifact?

Reply:
The reason for this moving is that very old stable version of **nssm** doesn't support the parameter **-version**. Meanwhile, this parameter is used by winpanda as a health check of package installation on the system. @anatolii-borysov please correct me if I'm wrong.

Reply:
This is not the only reason why do we use the latest "Featured pre-release nssm 2.24-101-g897c7ad (2017-04-26)" instead of the "Latest release nssm 2.24 (2014-08-31)". The former one brings a set of new features we need to implement service management on Windows  alike how it's done for Linux with systemd. Namely it's support for service event hooks which allows implementing pre/post service start/stop actions and which is absent in the nssm 2.24 (2014-08-31)